cmd/tetragon/main.go

```diff
@@ -246,24 +239,32 @@ func tetragonExecute() error {
 		go crd.WatchTracePolicy(ctx, observer.SensorManager)
 	}
 
-	var startSensors []*sensors.Sensor
+	// load base sensor
+	if err := base.GetInitialSensor().Load(ctx, observerDir, observerDir, option.Config.CiliumDir); err != nil {
```

Comment:
I'm scratching my head how this get compiled.. base.GetInitialSensor returns *sensors.Sensor and we return it as 'err' as error return value, what do I miss?

Reply:
`GetInitialSensor()` returns a `*sensors.Sensor`, which defines a `Load(...)` method that returns an error. So doing `GetInitialSensor().Load()` returns an error.

Reply:
aaargh there's .Load .. sorry for noise ;-)